src/__tests__/regression/test-cases.json

```diff
@@ -2,15 +2,13 @@
   "AVTqjPQGCCXim7Nl_gn3HMjE4k0Zi_eTFRJCNEVXZxw",
   "6eTVr8IKPNYbMHVcpHFXr-XNaL5hT6zRJXimcP-owmo",
   "mzvUgNc8YFk0w5K5H7c8pyT-FC5Y_ba0r7_8766Kx74",
-  "SJ3l7474UHh3Dw6dWVT1bzsJ-8JvOewtGoDdOecWIZo",
```

Comment:
note: these contracts fail on the SDK v1 because of the bug in the evolve implementation - https://discord.com/channels/357957786904166400/756557551234973696/881607008162304050